protocol/src/main/java/org/fido/iot/protocol/CryptoService.java

```diff
@@ -609,11 +641,23 @@ public void verifyMaroePrefix(Composite cose) {
    * @return True if the signature matches otherwise false.
    */
   public boolean verify(PublicKey verificationKey, Composite cose, Composite sigInfoA) {
-    if (null != sigInfoA && sigInfoA.size() > 0
-        && Arrays.asList(Const.SG_EPIDv10, Const.SG_EPIDv11, Const.SG_EPIDv20)
+    if (null != sigInfoA && Arrays.asList(Const.SG_EPIDv10, Const.SG_EPIDv11)
```

Comment:
Please add back the condition `&& sigInfoA.size() > 0`  here as well. As discussed for line 453 above.

Reply:
done